packages/core/src/routines/driver/ExecuteCommandRoutine.ts

```diff
@@ -87,41 +87,38 @@ export default class ExecuteCommandRoutine extends Routine<
     const isWatching = watchOptions.some((option) => {
       // Option
       if (option.startsWith('-')) {
-        const name = option.replace(/^-{1,2}/u, '');
-
-        // @ts-expect-error Allow this
-        return !!(args.options[name] || args.unknown[name]);
+        return !!context.getRiskyOption(option.replace(/^-{1,2}/u, ''));
       }
 
       // Param
       return args.params.includes(option);
     });
 
+    const outHandler = (chunk: Buffer) => {
+      console.log(String(chunk));
```

Comment:
any reason to not do `process.stdout.write(chunk)` instead?

Reply:
@hayes Because it disrupts the React rendering cycle. Console gets patched into it.